horace_core/sqw/coord_transform/@spher_proj_caption/spher_plot_titles_.m

```diff
@@ -39,10 +36,10 @@
 title = data.title;
 
 %uoff = data.uoffset;
-centre= data.uoffset;
-u_to_rlu = data.u_to_rlu;
+centre= data.offset;
+%u_to_rlu = data.proj.u_to_rlu;
```

Comment:
Debugging?

Reply:
this is where I am working now. Task incomplete